gapic/src/main/com/google/gapid/perfetto/views/SliceSelectionView.java

```diff
@@ -77,6 +80,29 @@ public SliceSelectionView(Composite parent, State state, SliceTrack.Slice slice)
       createLabel(main, slice.name);
     }
 
+    RenderStageInfo renderStageInfo = slice.getRenderStageInfo();
+    if (renderStageInfo != null) {
+      LinkedHashMap<String, String> props = new LinkedHashMap<String, String>();
+      if (renderStageInfo.frameBufferHandle != 0) {
+        props.put("VkFrameBuffer:", renderStageInfo.frameBufferName + "<" + String.format("0x%08X", renderStageInfo.frameBufferHandle) + ">");
```

Comment:
Put a space between the name and the `<`. Also, if the name is empty, don't show the brackets at all.